app/lib/actions/remote_execution/run_host_job.rb

```diff
@@ -17,7 +19,12 @@ def resource_locks
       end
 
       def plan(job_invocation, host, template_invocation, proxy_selector = ::RemoteExecutionProxySelector.new, options = {})
-        action_subject(host, :job_category => job_invocation.job_category, :description => job_invocation.description, :job_invocation_id => job_invocation.id)
+        features = template_invocation&.template&.remote_execution_features&.pluck(:label)&.uniq
```

Comment:
Is the safe navigation everywhere necessary? `template_invocation` should always be non-nil and have an associated template. `remote_execution_features` should always be defined on `template`

Reply:
Sure